tests/kafkatest/services/connect.py

```diff
@@ -105,14 +104,13 @@ def set_external_configs(self, external_config_template_func):
         """
         self.external_config_template_func = external_config_template_func
 
-    def listening(self, node):
+    def rest_loaded(self, node):
         try:
-            cmd = "nc -z %s %s" % (node.account.hostname, self.CONNECT_REST_PORT)
-            node.account.ssh_output(cmd, allow_fail=False)
-            self.logger.debug("Connect worker started accepting connections at: '%s:%s')", node.account.hostname,
-                              self.CONNECT_REST_PORT)
+            self.list_connectors(node)
+            self.logger.debug("Checking whether REST resources are loaded...")
```

Comment:
This debug message seems like it would appear _before_ we actually attempt to do any checking. It's probably worth keeping the old message (or something similar) _after_ the checking has been completed.

Reply:
Thanks, addressed.